WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -161,6 +161,15 @@
       patch :update, id: registration.id, registration: { status: :accepted }
       expect(registration.reload.accepted?).to eq true
     end
+
+    it "cannot accept own registration for a competition that isn't visible" do
+      competition.showAtAll = 0
+      competition.save!
```

Comment:
I think you _always_ run the validations, unless you have a real good reason not to!

Even if skipping validation is fine with the current code, that can change, silently turning this code into a bug.